tests/sentry/auth/test_helper.py

```diff
@@ -42,9 +42,10 @@ def setUp(self):
         self.auth_provider = AuthProvider.objects.create(
             organization=self.organization, provider=self.provider
         )
+        self.email = "test@example.com"
         self.identity = {
             "id": "1234",
-            "email": "test@example.com",
+            "email": self.email,
```

Comment:
Should we change this?

Reply:
Can you elaborate? I extracted this into an instance variable to reduce repetition of `self.identity["email"]` below.